addons/forum-toolbar/addon.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "Forum toolbar buttons",
+  "description": "Adds buttons in the forum editing toolbar to insert BBCode, such as [center] or [color].",
```

Comment:
This sounds like there are no buttons without this addon. Maybe "More forum toolbar buttons" and "... to insert more BBCode tags ..."?